backend/src/main/java/gov/cdc/usds/simplereport/db/repository/ResultRepository.java

```diff
@@ -0,0 +1,20 @@
+package gov.cdc.usds.simplereport.db.repository;
+
+import gov.cdc.usds.simplereport.db.model.Result;
+import gov.cdc.usds.simplereport.db.model.SupportedDisease;
+import gov.cdc.usds.simplereport.db.model.TestEvent;
+import gov.cdc.usds.simplereport.db.model.TestOrder;
+import java.util.List;
+
+public interface ResultRepository extends EternalAuditedEntityRepository<Result> {
+
+  List<Result> findAllByTestEvent(TestEvent testEvent);
```

Comment:
I know this might be a bit early to ask for, but i am sure well need to add indices for all these columns and possibly  multi-column indices. do we have a story for that?

Reply:
We do now! https://github.com/CDCgov/prime-simplereport/issues/3549